language/stdlib/modules/multi_account.move

```diff
@@ -0,0 +1,104 @@
+// Errors:
+// 0 -> ADDRESS_NOT_ALLOWED
+// 1 -> CURRENCY_NOT_HELD
+address 0x0:
+module MultiCurrencyHolding {
+    use 0x0::LibraAccount;
+    use 0x0::Transaction;
+    use 0x0::Libra;
+    use 0x0::Vector;
+
+    // A resource that specifies the addresses that are allowed to be added
+    // to this multi-currency holding.
+    resource struct ChildInfo {
+        allowed_addresses: vector<address>,
+    }
+
+    // Withdrawal capabilities for the added accounts, partitioned by their
+    // currency type.
+    resource struct AccountInfo<Token> {
+        withdrawal_caps: vector<LibraAccount::WithdrawalCapability>,
```

Comment:
This is perhaps a future modification, but might be useful for us to write a variant of this (or config in `ChildInfo`?) that lets children retain their `WithdrawalCapability`s (as we discussed before). That seems like a nice configuration: all deposits go to one address, but withdrawals come from the appropriate child to avoid seq # contention.

Reply:
Good point! For some reason this slipped my brain. I'll go ahead and add this in :)